docs/.vuepress/components/youtube-embed.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <iframe
+      :src="`https://www.youtube.com/embed${
+        type === 'playlist' ? '/videoseries?list=' : '/'
+      }${id}`"
+      frameborder="0"
+      allowfullscreen
+    >
+    </iframe>
+  </div>
+</template>
+
+<script>
+  export default {
+    props: [
```

Comment:
Can you add the prop types and if they are required or not?

Reply:
:+1: alright!